src/wallet/rpcwallet.cpp

```diff
@@ -5385,6 +5427,18 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
         throw JSONRPCError(RPC_HARD_FORK_DEPRECATION, shieldedPoolDeprecationErrorMessage);
     }
 
+    // we want to forbid unshielding to non-script addresses
+    if (chainActive.Height() + 1 >= GetUnshieldingToScriptOnlyForkHeight() &&
+        !fromTaddr && taddrRecipients.size() > 0)
+    {
+        for (const auto taddrRecipient: taddrRecipients)
```

Comment:
I believe we can avoid a copy here:
```suggestion
        for (const auto& taddrRecipient: taddrRecipients)
```

Reply:
Agree with you.
Modified accordingly.